src/app/components/unified-file-card/unified-file-card.component.html

```diff
@@ -34,10 +34,10 @@
                 <mat-icon>restore</mat-icon><ng-container i18n="Delete and redownload subscription video button">Delete and redownload</ng-container>
             </button>
             <button *ngIf="file_obj.sub_id && use_youtubedl_archive" (click)="emitDeleteFile(true)" mat-menu-item>
-                <mat-icon>delete_forever</mat-icon><ng-container i18n="Delete forever subscription video button">Delete forever</ng-container>
+                <mat-icon>delete_forever</mat-icon><ng-container i18n="Delete forever subscription video button">Delete and don't download again</ng-container>
```

Comment:
The original wording you used in this PR seemed a bit technical. A better UX wording would be to refer to this as "Delete and don't download again" like we discussed on Discord, so I felt free to commit this change myself. :) Same as below